src/build.sh

```diff
@@ -7,8 +7,13 @@ SRC=""
 #MODE="PROFILE"
 #MODE="MEMCHECK"
 MODE="RELEASE"
+VERSION_FILE="$(cd "$(dirname "$1")" || exit; pwd)/$(basename version.inc)"
```

Comment:
> Why not just `VERSION_FILE="version.inc"`?

Because when the `src/lib/compile.sh` script (which this `src/build.sh` script calls) is done compiling, it apparently does a `cd` to the `bin` to directory to call `wattsi` from there to emit the usage statement. So if we want to clean up (delete) the `version.inc` file when we’re done, and we otherwise don’t store the absolute path to it, then we’d need to use the relative path—`rm ../src/version.inc`—which to me seems uglier than just storing the absolute path.